examples/typescript/informer/informer.ts

```diff
@@ -0,0 +1,26 @@
+import * as k8s from '@kubernetes/client-node';
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+
+const listFn = (fn) => {
+    k8sApi.listNamespacedPod('default')
+        .then((res) => {
+            const podList = res.body;
+            fn(podList.items, podList.metadata!.resourceVersion);
+        })
+        .catch((err) => {
+            // tslint:disable-next-line:no-console
+            console.log(err);
+        });
+};
+
+const informer = k8s.makeInformer(kc, '/api/v1/namespaces', listFn);
```

Comment:
How come this isn't `/api/v1/namespaces/default/pods`?

Reply:
oops, fixed :)